packages/sdk/src/utils/math/tickHelpers.ts

```diff
@@ -139,3 +140,12 @@ function tickToFixedRate(tick: number) {
 
   return priceToFixedRate(price);
 }
+
+export function fixedRateToPrice(fixedRate: number): string {
```

Comment:
wonder if some of this tick math that's shared across v1 and v2 vamms can be abstracted away into commons type of package?